ghost/api-framework/lib/headers.js

```diff
@@ -122,28 +122,24 @@ module.exports = {
         }
 
         const locationHeaderDisabled = apiConfigHeaders?.location === false;
-        const hasFrameData = ['add', 'copy'].includes(frame?.method) && result[frame.docName]?.[0]?.id;
+        const hasLocationResolver = apiConfigHeaders?.location?.resolve;
+        const hasFrameData = (frame?.method === 'add' || hasLocationResolver) && result[frame.docName]?.[0]?.id;
```

Comment:
Yessss! No additional hardcoded method in the framework layer and gives us a clear path of migration to move away from hardcoded `add` when we find appetite for it 👍